Silicon/ArrowlakePkg/Library/PcieClientRpLib/PcieClientRpLib.inf

```diff
@@ -0,0 +1,42 @@
+## @file
+# Component description file for the PcieClientRpLib
+#
+#  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+
+[Defines]
+  INF_VERSION = 0x00010017
+  BASE_NAME = PcieClientRpLib
```

Comment:
This lib is under ARL silicon, not necessary to have "client" in its name. Better rename the folder and file name to remove Client.
If this library could be common for other platforms, it would be put to Silicon\CommonSocPkg folder.

Reply:
Renamed the folder and removed client from filename in latest push